docs/pages/versions/unversioned/sdk/video.mdx

```diff
@@ -294,6 +294,21 @@ import { createVideoPlayer } from 'expo-video';
 const player = createVideoPlayer(videoSource);
 ```
 
+### Caching videos
+
+If your app frequently replays the same video, caching can be utilized to minimize network usage and enhance user experience, albeit at the cost of increased device storage usage.
+`expo-video` supports video caching on `Android` and `iOS` platforms. This feature can be activated by setting the [`useCaching`](#videosource) property of a [`VideoSource`](#videosource) object to `true`.
+The cache is persistent and will be cleared on a least-recently-used basis once the preferred size is exceeded. Furthermore, the system can clear the cache due to low storage availability, so it's not advisable to depend on the cache to store critical data.
+The cache functions offline. If a portion or the entirety of a video is cached, it can be played from the cache even when the device is offline until the cached data is exhausted.
+
+> The cache cannot be used with HLS video sources and DRM-protected videos.
```

Comment:
Based on the  following note below: 
![CleanShot 2025-01-06 at 13 50 28@2x](https://github.com/user-attachments/assets/7c130abe-a4d9-4aed-be68-a9030ad0146c)

I think we should keep the same verbiage here as well. Right now, it is confusing to see the same message in two places, with the difference being the iOS platform.

I think it'll be good to say here as well that: "Due to platform limitations, the cache cannot be used with HSL video sources on iOS. Caching DRM-protected videos is not supported on Android and iOS."

If this is technically right, can we also update the "Caching DRM-protected videos..." note in the type definition to mention that it is not supported on any platform?